src/python/pants/rules/core/test_test.py

```diff
@@ -145,15 +145,13 @@ def make_debug_request(target):
         MockGet(
           product_type=BuildFileAddress,
           subject_type=BuildFileAddresses,
-          mock=lambda tgt: BuildFileAddress(
-            rel_path=f'{tgt.spec_path}/BUILD', target_name=tgt.target_name,
-          )
+          mock=lambda addresses: addresses.dependencies[0]
         ),
       ],
     )
 
     self.assertEqual(1, res.exit_code)
-    self.assertEquals(console.stdout.getvalue(), dedent("""\
+    self.assertEqual(console.stdout.getvalue(), dedent("""\
```

Comment:
Would it be useful to put up a separate PR fixing all of these at once?

Reply:
Likely. I try to restrict myself from making too many changes like this (f-strings, using `assert` instead of `self.assertX`, etc) but let myself fix this because it was right there and a tiny change.